lib/telephone_appointments/response.rb

```diff
@@ -0,0 +1,17 @@
+module TelephoneAppointments
+  class Response
+    def initialize(response)
+      @response = response
```

Comment:
Can we add a private reader for accessing this elsewhere internally to this class?